app/controllers/export_controller.rb

```diff
@@ -70,9 +70,19 @@ def progress
   def status
     map = Map.find(params[:id])
     if export = map.export
-      render json: export.to_json
+      if export.export_url.present?
+        status_response = ExporterClient.new(export.export_url).status
+        render json: status_response
```

Comment:
Then we can return the whole `status.json` here.

Reply:
I am a little confused about the 2 requested changes, because `status` is just a method written in the `ExporterClient` to retrieve the body of the response as JSON. So I think it is already rendering the whole thing, right?

Reply:
Ah, i see! Great. Then these are all good. Sorry, it's confusing because `status` (in the old Exports model) was also a field storing just a string like `complete`, so I got confused. I see this is the whole JSON object. Perfect!